zh/enable-tls-between-components.md

```diff
@@ -1058,9 +1058,9 @@ category: how-to
 
     参考官网文档：[Download TiDB installation package](https://pingcap.com/docs/v3.0/reference/tools/pd-control/#download-tidb-installation-package)。
 
-2. 连接集群。
+2. 下载 Client 端证书。
```

Comment:
@weekface PTAL if my modification is correct. Thanks!

Reply:
good catch